WorkloadManagementSystem/DB/PilotAgentsDB.py

```diff
@@ -19,14 +19,15 @@
 
 __RCSID__ = "$Id$"
 
+from types import IntType, LongType, ListType
```

Comment:
Can you use isinstance instead ?

Reply:
I didn't change this file as changes like the one you're pointing out now are already in the integration branch.